lib/unicore/mktables

```diff
@@ -10049,17 +10058,28 @@ sub finish_property_setup($file) {
     # file directly (it was documented in 5.12 and 5.14 as being thusly
     # usable), keep it from being adjusted.  (range_size_1 is
     # used to force the traditional format.)
-    if (defined (my $nfkc_cf = property_ref('NFKC_Casefold'))) {
-        $nfkc_cf->set_to_output_map($EXTERNAL_MAP);
-        $nfkc_cf->set_range_size_1(1);
+    foreach my $property (qw(NFKC_Casefold NFKC_Simple_Casefold)) {
+        if (defined (my $cf = property_ref($property))) {
+            $cf->set_to_output_map($EXTERNAL_MAP);
+            $cf->set_range_size_1(1);
+        }
     }
+
     if (defined (my $bmg = property_ref('Bidi_Mirroring_Glyph'))) {
         $bmg->set_to_output_map($EXTERNAL_MAP);
         $bmg->set_range_size_1(1);
     }
 
     property_ref('Numeric_Value')->set_to_output_map($OUTPUT_ADJUSTED);
 
+    # These two properties have no short names and the file names for them
+    # clash in DOS 8.3.  Work around this by creating shorter file names that
```

Comment:
Where are we still limited by 8.3?

Reply:
On IRC the other day, I asked if we were still limited, and the answer was yes.

Reply:
For unicode filenames yes, but for ASCII filenames we don't AFAIK.

Reply:
I'd prefer to leave this as-is, since it is trivial to do, just in case.  And I have WIP which should get rid of them altogether.